content/_code-samples/escrow/py/return_escrow_sequence.py

```diff
@@ -0,0 +1,21 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import Tx
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # Connect to the testnetwork
+
+prev_txn_id = ""
```

Comment:
I'd add a check here printing out that there's a missing input if this is still empty when run. 

Ex. 

```
prev_txn_id = ""
if(prev_txn_id == ""):
    print("No transaction id provided. Use create_escrow.py to generate an escrow transaction, then you can look it up by modifying prev_txn_id to use that transaction's id.")
```

Reply:
(Otherwise when someone runs it for the first time, they may not realize what they have to do in order to make use of the snippet)

Reply:
ok